gossip/src/crds.rs

```diff
@@ -316,10 +322,11 @@ impl Crds {
                 } else if matches!(route, GossipRoute::PushMessage(_)) {
                     let entry = entry.get_mut();
                     if entry.num_push_recv == Some(0) {
-                        self.stats.lock().unwrap().num_redundant_pull_responses += 1;
+                        stats.num_redundant_pull_responses += 1;
                     }
                     let num_push_dups = entry.num_push_recv.unwrap_or_default();
                     entry.num_push_recv = Some(num_push_dups.saturating_add(1));
+                    stats.num_duplicate_push += 1;
```

Comment:
if `entry.num_push_recv == Some(0)` in line 324, then technically this is not a duplicate push, no?
I mean in that case the issue is redundant pulls, not duplicate push, so probably shouldn't count as duplicate push.

Reply:
ya i went back and forth on this one. Think it just depends on definition of duplicate push. Since it technically is a duplicate message that we already have. But if we're looking specifically at receiving a push and then receiving another push with the same `CrdsValue` then you are correct. I'm ok treating it as the latter (aka what you are suggesting)! Since we can always find all duplicates by adding `num_redundant_pull_responses` and `num_duplicate_push`